server/internal/tasks/index.ts

```diff
@@ -2,42 +2,94 @@ import droplet from "@drop-oss/droplet";
 import type { MinimumRequestObject } from "~/server/h3";
 import aclManager from "../acls";
 
-/**
- * The TaskHandler setups up two-way connections to web clients and manages the state for them
- * This allows long-running tasks (like game imports and such) to report progress, success and error states
- * easily without re-inventing the wheel every time.
- */
-
-type TaskRegistryEntry = {
+import cleanupInvites from "./registry/invitations";
+import cleanupSessions from "./registry/sessions";
+import checkUpdate from "./registry/update";
+import cleanupObjects from "./registry/objects";
+import { taskGroups, type TaskGroup } from "./group";
+
+// a task that has been run
+type FinishedTask = {
```

Comment:
We should probably keep historic tasks in the database, perhaps with some sort of 'log rotation'?

Reply:
Yea, i was probably just gonna create a new task to cleanup logs older then x days